tests/security/authz_deny/scripts/authz_deny.txt

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
```

Comment:
I'm in the process of trying to clean up all the current tests to put the entire script into a single .sh file (instead of the current one or more .txt files), and with one simple boilerplate .go file (which hopefully in the future will be generated).

It would be good if this new test follows that convention (i.e., rename authz_deny.txt to authz_deny.sh). Note that you will need to add a shellcheck comment on line 2 and put quotes around the snip.sh path, to make lint happy. Here's an example of the pattern I'm trying to make all tests follow: https://github.com/istio/istio.io/blob/master/tests/trafficmanagement/scripts/request_routing.sh

Reply:
> This looks great! It make me wonder if we even need the _verify_xxx functions. Maybe we should always use run_and_verify?

That thought occurred to me, too, but I didn't want to be too presumptuous.  There were a couple places that seemed to use _verify functions that didn't invoke a command.  Now that I know that you like the approach, I'll look into that more option more seriously.
 
> Regarding lint-fast, if it's not easy to fix it, should we change the README to use make lint? If it's not too slow, it's not a big deal.

I'll see if I can track down what changed.

I'll take care of changing the name of the script to "authz_deny.sh".   I like that better, too.

Reply:
When you're looking at existing tests to see how they use _verify functions, keep in mind that only the tests that have been converted to .sh files are really what I think "correct" tests. The other ones are just a first pass to get rid of the old snippet gen stuff, so I could remove it from the underlying framework. I'm going to keep cleaning up tests, as much as I find time, so hopefully in a few days they will be all cleaned up single .sh tests. At that point, I doubt that there will be any verify calls that don't also need to call a function, but you never know